<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbCheckDialog</class>
 <widget class="QDialog" name="DbCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This dialog offers some functions to cleanup the data in Nepomuk.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="KPushButton" name="checkCollection">
       <property name="toolTip">
        <string>Find all Collections that don't have an article connected to it.</string>
       </property>
       <property name="text">
        <string>SyncData Check</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="KPushButton" name="checkReference">
       <property name="toolTip">
        <string>Finds all reference that have no publication attached to it. This should never happen and can be deleted.</string>
       </property>
       <property name="text">
        <string>Reference Check</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="KPushButton" name="checkSeries">
       <property name="toolTip">
        <string>Finds all Series that are not used anywhere.</string>
       </property>
       <property name="text">
        <string>Series Check</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="KPushButton" name="checkAuthor">
       <property name="toolTip">
        <string>Finds all authors that are not in an akonadi collection or used somewhere else.</string>
       </property>
       <property name="text">
        <string>Author Check</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="KPushButton" name="checkAll">
       <property name="toolTip">
        <string>Lists all publication data created from this program so the Nepomuk database can be cleaned completly.</string>
       </property>
       <property name="text">
        <string>Clear Database</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="KPushButton" name="checkDocumentPart">
       <property name="toolTip">
        <string>Finds all Document parts (chapters and such) that are not attached to a publication.</string>
       </property>
       <property name="text">
        <string>DocumentPart Check</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="KPushButton" name="checkTags">
       <property name="text">
        <string>Clean Tags</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KPushButton" name="removeData">
       <property name="toolTip">
        <string>Starts to remove the resource found by the functions above.</string>
       </property>
       <property name="text">
        <string>Remove Data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DbCheckDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DbCheckDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
